### ISSUE TITLE
Powder migration outside of GeomOpt

### ISSUE THREAD

**Issue by fredericpoitevin** [apeck12/sfx_utils#26]

**Comment by apeck12** [comment]
Specifically, migrate to `RunDiagnostics`. It probably makes sense to save the average and standard deviation powders as well in case we want to offer a way to build masks à la DIALS: According to the documentation in their [mask generator](https://github.com/cctbx/cctbx_project/blob/ade0687066f1205f3650ecfadc7ac0d7104a3dde/xfel/command_line/make_dials_mask.py):
```
# This code reads three images and builds a mask from
# them.  The first image should be an average from a dark run, the second the
# standard deviation from that run.  The third image should be a maximum projection
# from a run with the beam on.
```

**Comment by apeck12** [comment]
Resolved by PR #32
